Added coriander chicken recipe (#534)
LukeSmithxyz/based.cooking: src/coriander-chicken.md

# Coriander Chicken

Boneless chicken marinated in a gravy of yoghurt and coriander

- ⏲️ Prep time: 30 min
- 🍳 Cook time: 20 min
- 🍽️ Servings: 3

## Ingredients

> - 500 gm boneless chicken breast
> - 250 gm fat free yoghurt
> - 250 gm coriander leaves

Reviewer: ```suggestion
- 500g boneless chicken breast
- 250g fat free yoghurt
- 250g coriander leaves
```

- Mustard oil / Sunflower oil
- Sugar
- Ginger
- Garlic
- 2 medium onions
- Green chillies
- Garam masala
- 2/3 Cardamom pods
- 1 Bayleaf
- 1 Cinnamom stick

## Directions

1. Cut the boneless chicken into small bite sized pieces.

> 2. In a mixer, put 250 gm coriander leaves, 2/3 green chillies, 250 gm yoghurt, 20 gm ginger, 9/10 cloves of garlic, 2 medium sized onions and mix well.

Reviewer: ```suggestion
2. In a mixer, put 250g coriander leaves, 2/3 green chillies, 250g yoghurt, 20g ginger, 9/10 cloves of garlic, 2 medium sized onions and mix well.
```

> 3. Add the green chillies according to the your taste.

Reviewer: ```suggestion
3. Add the green chillies according to your taste.
```

> 4. Pour half of the mixture from step 2 into a bowl and mix it thouroughly with the chicken.

Reviewer: ```suggestion
4. Pour half of the mixture from step 2 into a bowl and mix it thoroughly with the chicken.
```

5. Into this chicken mixture add 1 tbsp Garam Masala and salt to taste.
6. Keep the resulting marinated chicken aside for about 30 minutes.
7. In a frying pan add 3 tbsp of either mustard oil or sunflower oil.
8. Once the oil is hot, add 1 tbsp sugar and cook until the sugar melts into the oil.
9. Add 2/3 cardamom pods, 1 small cinnamon stick, and 1 bayleaf into the hot oil.
10. Simmer for aboout 1 minute, then lower the heat to medium.
11. On medium heat, add the marinated chicken mixture and cook till oil starts separating from the chicken.
12. Add salt to taste and keep stirring the chicken to make sure it does not burn or stick to the bottom of the pan.
13. Once the oil separates, add the remaining mixture which was left behind after step 4.

> 14. Stir until it has a thick consistemncy. Serve with [naan](naan-bread.html) or [rice](rice.html).

Reviewer: ```suggestion
14. Stir until it has a thick consistency. Serve with [naan](naan-bread.html) or [rice](rice.html).
```

## Contribution

> - Chetan Basuray - [website](https://github.com/chetanbasuray)
> - Chetan Basuray - btc: `199H7ee5t3D2N1fnpUmFEqfVMqe3aq4eMr`
> - Chetan Basuray - eth: `0x911f88ac1e699f111a22a1b4049e44ef8e8b08cb`
> - Chetan Basuray - monero: `84PTQNJeyUuTVHN3daWwmPLBCL5GLt4WB14e2ZqFMu5kiPZDgst5hVUPVjVmdJJCvpPdi3Tt6KU853JxSQY3xCHPUtLECbt`

Reviewer: ```suggestion
- Chetan Basuray
    - [website](https://github.com/chetanbasuray)
    - btc: `199H7ee5t3D2N1fnpUmFEqfVMqe3aq4eMr`
    - eth: `0x911f88ac1e699f111a22a1b4049e44ef8e8b08cb`
    - monero: `84PTQNJeyUuTVHN3daWwmPLBCL5GLt4WB14e2ZqFMu5kiPZDgst5hVUPVjVmdJJCvpPdi3Tt6KU853JxSQY3xCHPUtLECbt`
```
